.gitignore

```diff
@@ -32,6 +32,7 @@ mydatabase
 celerybeat-schedule*
 node_modules
 *.jaseci_logs/
+_jaseci_ai_kit
```

Comment:
Is this needed?

Reply:
actually, I have a backup folder that has some of the bi_encoder results.